src/sonic-config-engine/minigraph.py

```diff
@@ -457,9 +456,10 @@ def parse_xml(filename, platform=None, port_config_file=None):
         ports.setdefault(port_name, {})['speed'] = port_speeds_default[port_name]
 
     for port_name in port_speed_png:
-        # if port_name is not in port_config.ini, still consider it.
-        # and later swss will pick up and behave on-demand port break-up.
-        # if on-deman port break-up is not supported on a specific platform, swss will return error.
+        # not consider port not in port_config.ini
+        if port_name not in ports:
```

Comment:
>if port_name not in ports: [](start = 8, length = 26)

Should we treat this as a minigraph generator's bug instead of parser's bug?

Reply:
if we can mitigate that in minigraph, it should be ideal

Reply:
Discussed with Guohan, this is the good.

---
In reply to: [189410214](https://github.com/Azure/sonic-buildimage/pull/1727#discussion_r189410214) [](ancestors = 189410214)